data/mods/More_Survival_Tools/items.json

```diff
@@ -20,8 +19,47 @@
     "turns_per_charge": 0,
     "ammo": "tinder",
     "revert_to": "null",
+    "use_action": {
+        "type": "transform",
+        "msg": "You light the tinder.",
+        "target": "hobo_stove_on",
+        "active": true,
+        "need_fire": 1,
+        "need_fire_msg": "You need a lighter or fire to light this."
+    }
+  },
+  {
+    "id": "hobo_stove_on",
```

Comment:
If any light at all would escape from the stove, it should have ```"flags": "LIGHT_3",``` or so.  a value of 3 won't make it a light source per se, but it will make it visible at night.  A value of 4-6 will make it shed a little light, probably shouldn't be higher than 6.